gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ocr2/inspection/inspectResponses.ts

```diff
@@ -1,11 +1,25 @@
 import { Result } from '@chainlink/gauntlet-core'
+import { EventParser, BorshCoder, Idl, Event } from '@project-serum/anchor'
 import { SolanaCommand, TransactionResponse } from '@chainlink/gauntlet-solana'
 import { PublicKey } from '@solana/web3.js'
 import { CONTRACT_LIST, getContract } from '../../../../lib/contracts'
-import { inspection } from '@chainlink/gauntlet-core/dist/utils'
+import { inspection, logger, BN } from '@chainlink/gauntlet-core/dist/utils'
+import { ORACLES_MAX_LENGTH } from '../../../../lib/constants'
 
 type Input = {}
 
+type NewTransmission = {
```

Comment:
Let's group types like this not under commands that depend on them, but as general OCR2  types somewhere like `contracts/ocr2/types.ts`.

Long term we'll need to think about the overall structure of our TS toolchain (SDK + Gautlet). One of the suggestions would be to move types like this and core functionality to our TS SDK (`@chainlink/solana-sdk`) and then build Gauntlet on top of it.

cc @RodrigoAD